docs/query_execution_plan_in_TiSpark.md

```diff
@@ -0,0 +1,84 @@
+```
+
+Only the node of `RegionTaskExec` and the child node `CoprocessorRDD`(`FetchHandleRDD`) and are the output of TiSpark and the rest node is the output of Spark. So here we mainly explain the `RegionTaskExec` and `CoprocessorRDD`(`FetchHandleRDD`).
+
+## Understand EXPLAIN output in `CoprocessorRDD`(`FetchHandleRDD`)
+
+### Operator that perform table scans
+
+An operator is a particular step that is executed as part of returning query results. The operators that perform table scans (of the disk or the TiKV Block Cache) are listed as follows:
+
+* **`TableRangeScan`**: Table scans with the specified range. We consider full table scan as a special case of `TableRangeScan`, so full table scan is also called `TableRangeScan`.
+* **`TableRowIDScan`**: Scans the table data based on the `RowID`. Usually follows an index read operation to retrieve the matching data rows.
+* **`IndexRangeScan`**: Index scans with the specified range. We consider full index scan as a special case of `IndexRangeScan`, so full index scan is also called `IndexRangeScan`.
+
+### Operator that aggregates the data from TiKV/TiFlash
+
+TiSpark aggregates the data or calculation results scanned from TiKV/TiFlash. The data aggregation operators can be divided into the following categories:
+
+- **`TableReader`**: Aggregates the data obtained by the underlying operator `TableRangeScan` in TiKV.
+- **`IndexReader`**: Aggregates the data obtained by the underlying operator `IndexRangeScan` in TiKV.
+- **`IndexLookUp`**: First aggregates the RowIDs (in TiKV) scanned by the first scan in the index. Then at the second scan in the table, accurately reads the data from TiKV based on these RowIDs. At the first scan in the index, there is  `IndexRangeScan` operator; at the second scan in the table, there is the `TableRowIDScan` operator.
+
+### `Range`&`RangeFilter`
+
+In the `WHERE`/`HAVING`/`ON` conditions, the TiSpark optimizer analyzes the result returned by the primary key query or the index key query. For example, these conditions might include comparison operators of the numeric and date type, such as `>`, `<`, `=`, `>=`, `<=`, and the character type such as `LIKE`.
+
+The `Range` in `CoprocessorRDD`(`FetchHandleRDD`) represents the range of scanning. `RangeFilter` indicates which expression(s) the range is made up of. If `RangeFilter` is empty, it indicates a full table scan or full index scan. `RangeFilter` generally appears when the query involves an index range, when query the expressions in the `RangeFilter` form the scanned range from left to right.
```

Comment:
> `RangeFilter` generally appears when the query involves an index range

how about pure TableRangeScan?

Reply:
> > `RangeFilter` generally appears when the query involves an index range
> 
> how about pure TableRangeScan?
It will appear in `TableRangeScan` when the query involves cluster index